src/main/java/com/connexta/store/controllers/StoreController.java

```diff
@@ -106,14 +104,21 @@
     return ResponseEntity.created(location).build();
   }
 
-  /** TODO Add tests for this endpoint. */
   @Override
   public ResponseEntity<Void> addMetadata(
       final String acceptVersion,
       @Pattern(regexp = "^[0-9a-zA-Z]+$") @Size(min = 32, max = 32) final String productId,
       @Pattern(regexp = "^[0-9a-zA-Z\\-]+$") @Size(min = 1, max = 32) final String metadataType,
       @Valid final MultipartFile file) {
-    // TODO validate params
+    final String expectedAcceptVersion = storeApiVersion;
+    if (!StringUtils.equals(acceptVersion, expectedAcceptVersion)) {
```

Comment:
Our API says we should accept previous minor versions, does this need to be updated to account for that?

Reply:
We don't currently support that even though we should. See the message in the exception.